boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuflpr.yaml

```diff
@@ -14,4 +14,3 @@ supported:
   - gpio
   - i2c
   - spi
-  - watchdog
```

Comment:
If I understand the commit message correctly, the issue is missing `watchdog0` alias. So why not add it instead of disabling support?

Reply:
Overlay file for FLPR is also needed in at least two samples/tests.
I don't have experience on using FLPR/PPR to check if watchdog driver works on RISC V.

Reply:
Adding this overlay:
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/drivers/watchdog/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay
but called `nrf54l15pdk_nrf54l15_cpuflpr.overlay` fixes the problem on my side.

Reply:
OK. I reverted removal of watchdog from the supported peripherals list for nrf54l15/cpuflpr. I added overlay file as suggested.